Makefile

```diff
@@ -21,12 +21,13 @@ HG_BIN := $(shell command -v $(HG_BIN_NAME) 2> /dev/null)
 DOCKER_COMPOSE_BIN := $(shell command -v $(DOCKER_COMPOSE_BIN_NAME) 2> /dev/null)
 DOCKER_BIN := $(shell command -v $(DOCKER_BIN_NAME) 2> /dev/null)
 
-GLIDE_VERSION := $(shell ${GLIDE_BIN} --version | sed -rn 's/([^0-9.]*)([0-9]*\.[0-9]*)(.*)/\2/p')
+#GLIDE_VERSION := $(shell $(GLIDE_BIN) --version | sed -rn 's/([^0-9.]*)([0-9]*\.[0-9]*)(.*)/\2/p')
+GLIDE_VERSION := 0.13
```

Comment:
The point of this code was to test the active version of Glide was at least version 0.12 or higher.
The code was requested by @xcoulon in the original PR; unfortunately, although Glide makes it easy to ensure proper versions of Go code are used in the build, there's no way to ensure the proper version of Glide is used.

I'm fine with removing the test altogether (we don't test for other tool versions), but hardcoding GLIDE_VERSION effectively neuters the test - so I'd suggest getting rid of it (and not adding bc to the image).

Reply:
hehe yea, I'm aware. This was just an attempt at debugging where the "exception" in the build log was coming from..

```
/bin/sh: --: invalid option
Usage:	/bin/sh [GNU long option] [option] ...
	/bin/sh [GNU long option] [option] script-file ...
GNU long options:
	--debug
	--debugger
	--dump-po-strings
	--dump-strings
	--help
	--init-file
	--login
	--noediting
	--noprofile
	--norc
	--posix
	--protected
	--rcfile
	--rpm-requires
	--restricted
	--verbose
	--version
Shell options:
	-irsD or -c command or -O shopt_option		(invocation only)
	-abefhkmnptuvxBCHP or -o option
```